### PR TITLE
Add extension method to get chirality of hand

### PR DESCRIPTION
**Pull request by Dwarph**
## Summary

Small addition to the extension methods to get the chirality of a hand. I use this all the time & thought it was about time it got into the plugin!

## Contributor Tasks

_These tasks are for the merge request creator to tick off when creating a merge request._

- [x] Pair review with a member of the QA team.
- [x] Add any release testing considerations to the MR for the next release.
- [x] Check any relevant CHANGELOG files have been updated.
- [x] Ensure documentation requirements are met e.g., public API is commented.
- [x] Consider any licensing/other legal implications for this MR e.g. notices required by any new libraries.
- [x] Add any relevant labels such as `breaking` to this MR.
- [ ] If this MR closes a Jira issue, make sure the fix version on the JIRA issue is set to the correct one.

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

[Use emojis in review threads to communicate intent and help contributors.](https://github.com/ultraleap/UnityPlugin/blob/develop/CONTRIBUTING.md#review-threads)

- [x] Code reviewed.
- [x] Non-code assets e.g. Unity assets/scenes reviewed.
- [x] Documentation has been reviewed. Includes checking documentation requirements are met and not missing e.g., public API is commented.
- [x] Checked and agree with release testing considerations added to MR for the next release.

## Closes JIRA Issue

Closes UNITY-960

### CODE REVIEW

File: Packages/Tracking/Core/Runtime/Scripts/Utils/HandUtils.cs [hunk 1]
```diff
@@ -379,6 +379,13 @@ public static float GetFingerStrength(this Hand hand, int finger)
             return Vector3.Dot(hand.Fingers[finger].Direction, -hand.DistalAxis()).Map(-1, 1, 0, 1);
         }
 
+        /// <summary>
+        /// Returns the Chirality of the hand
+        /// </summary>
+        public static Chirality GetChirality(this Hand hand) {
```
Comment (MattGrayUL) [inline Packages/Tracking/Core/Runtime/Scripts/Utils/HandUtils.cs:385]: can this brace go on the next line to match the rest of the file please? :)

Comment (Dwarph) [reply Packages/Tracking/Core/Runtime/Scripts/Utils/HandUtils.cs]: done

**Review by MattGrayUL — APPROVED** [review]
I am happy with this, it provides the chirality of a hand when requested

I tested this in editor with a custom script that requested the chirality of each hand

I logged the result of this utility as well as the known '.Left' result and it matched :)

**Comment by MattGrayUL** [comment]
@Greg-UL if you're happy with my comments, we can merge :)

**Comment by Greg-UL** [comment]
Testing seems on point. 

Although what if someone had two left hands?

**Comment by MattGrayUL** [comment]
> Testing seems on point.
> 
> Although what if someone had two left hands?

Currently not possible as we max out at 1 of each per Frame, we would have the same issue with the existing '.IsLeft' so is not a _new_ issue if the max number of matching chirality hands ever changed

**Comment by Greg-UL** [comment]
It was initially a joke but i did consider multi device and multiplayer and the impact this would have.

**Pull request merged**